docs/en/pid-data-model.rst

```diff
@@ -393,8 +395,295 @@ The combined format for the PID issuance is given by
+
+The the data elements contained in the document, uses the same namespace 
+for the required PID attributes, while the national PID attributes use the domestic
+namespace `eu.europa.ec.eudiw.pid.it.1`, defined in this implementation profile.
+
+.. note::
+
+    This specification publishes the Italian PID MDOC-CBOR namespace, 
+    including all  data element identifiers, their definition 
+    and encoding format. 
+
+
+MDOC-CBOR Data Schema
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+A PID issued in the MDOC-CBOR format must have the following data element:
+
+.. list-table:: 
+    :widths: 20 20 60
+    :header-rows: 1
+
+    * - **Element**
+      - **Encoding**
+      - **Description**
+    * - **docType**
+      - tstr (text string)
+      - The document type. 
+    * - **issuerSigned**
+      - bstr (byte string)
+      - The issuerSigned object containing the `nameSpaces` and the 
+        `issuerAuth` objects.
+
+.. note::
+    The `issuerSigned` contains the `nameSpaces` object and the 
+    `Mobile Security Object`, however sole signed object is
+    the `Mobile Security Object`, while the `nameSpaces` object is not
+    signed.
+
+
+A non-normative example of the PID document is represented below.
+
+..code-block::
+
+  {
+    "docType": "eu.europa.ec.eudiw.pid.1",
+    "issuerSigned": {
+        "nameSpaces": {
+            "eu.europa.ec.eudiw.pid.1": [
+                {
+                    "digestID": 0,
+                    "random": $binary-random-value,
+                    "elementIdentifier": "family_name",
+                    "elementValue": "Rossi"
+                },
+                {
+                    "digestID": 1,
+                    "random": $binary-random-value,
+                    "elementIdentifier": "given_name",
+                    "elementValue": "Mario"
+                },
+                {
+                    "digestID": 2,
+                    "random": $binary-random-value,
+                    "elementIdentifier": "birth_date",
+                    "elementValue": {
+                        "CBORTag:1004": "2007-10-20"
+                    }
+                },
+                {
+                    "digestID": 3,
+                    "random": $binary-random-value,
+                    "elementIdentifier": "birth_place",
+                    "elementValue": "Rome"
+                },
+                {
+                    "digestID": 4,
+                    "random": $binary-random-value,
+                    "elementIdentifier": "birth_country",
+                    "elementValue": "IT"
+                }
+            ],
+            "eu.europa.ec.eudiw.pid.it.1": [
+                {
+                    "digestID": 0,
+                    "random": $binary-random-value,
+                    "elementIdentifier": "tax_id_code",
+                    "elementValue": "TINIT-XXXXXXXXXXXXXXXX"
+                },
+            ],
+            ]
+        },
```

Comment:
Should we be consistent on claims regardless of data format? In https://italia.github.io/eidas-it-wallet-docs/en/pid-data-model.html we used `tax_id_number`, `place_of_birth`, etc.

Reply:
I fully agree with you but unfortunately the PID rulebook is stucked on the defined attributes and approaches of ISO 18013-5

regarding `tax_id_code` vs `tax_id_number` we found a typo @fmarino-ipzs 
in a section is defined with a name and in other with the other name

Reply:
I vote for `code` since is it contains both numbers and letters

Reply:
typo resolved here https://github.com/italia/eudi-wallet-it-docs/pull/64

@fmarino-ipzs @balanza ^